.expeditor/verify.pipeline.yml

```diff
@@ -665,8 +519,10 @@ steps:
       HAB_STUDIO_SECRET_HAB_LICENSE: "accept-no-persist"
     command:
       - .expeditor/scripts/verify/build_component.sh backline
-    agents:
-      queue: 'default-privileged'
+    expeditor:
+      executor:
+        linux:
```

Comment:
Can these `build_component` verifications run in a docker executor?  Does it matter?

Reply:
Possibly? It wasn't before so I didn't want to rock the boat in this PR, it's big enough. If you want to test it in a followup, that would be :awesome: